src/main/java/com/puppycrawl/tools/checkstyle/filters/SuppressionXpathFilter.java

```diff
@@ -379,32 +379,33 @@
  * &lt;/module&gt;
  * </pre>
  * <p>
- * A <a href="/dtds/suppressions_1_2_xpath_experimental.dtd"><em>suppressions XML document</em></a>
+ * A <a href="https://checkstyle.org/dtds/suppressions_1_2_xpath_experimental.dtd"><em>
+ * suppressions XML document</em></a>
  * contains a set of {@code suppress} and {@code suppress-xpath} elements,
  * where each {@code suppress-xpath} element can have the following attributes:
  * </p>
  * <ul>
  * <li>
- * {@code files} - a <a href="property_types.html#regexp">Regular Expression</a>
+ * {@code files} - a <a href="https://checkstyle.org/property_types.html#regexp">Regular Expression</a>
```

Comment:
This line is now longer than 100 characters.
The pattern should be something like `href=&quot;[^&quot;]&gt;`
I'll start a new issue for this.